src/common/services/casApi.service.js

```diff
@@ -0,0 +1,69 @@
+import angular from 'angular';
+import 'angular-environment';
+import {Observable} from 'rxjs/Observable';
+import 'rxjs/add/observable/from';
+import 'rxjs/add/operator/map';
+import 'rxjs/add/operator/do';
+
+let serviceName = 'casApiService';
+
+class CasApi {
```

Comment:
How much is this going to be used? If it is for a couple of requests, do you think it would be better to do them directly in the session service? There might be many reasons we are going to CAS though. It just seems like a lot of duplicate code from the cortex api service to just change the url prefix.